datarepo-clienttests/src/main/java/scripts/testscripts/BillingProfileOwnerHandoffTest.java

```diff
@@ -40,13 +40,11 @@ public void userJourney(TestUserSpecification testUser) throws Exception {
     try {
       profile = ownerUser1Api.createProfile(billingAccount, "profile_permission_test", true);
       String profileId = profile.getId();
-      // Remove stewards from the owner list. Otherwise, voldemort has access by default :(
-      ownerUser1Api.deleteProfilePolicyMember(profileId, "owner", stewardsEmail);
 
       ownerUser1Api.addProfilePolicyMember(profileId, "owner", userUser.userEmail);
-      userUserApi.deleteProfilePolicyMember(profileId, "owner", ownerUser1.userEmail);
+      userUserApi.deleteProfilePolicyMember(profileId, "owner", ownerUser2.userEmail);
       // Make sure removed owner can perform none of the operations
-      testOperations(ownerUser1Api, RoleState.NONE, profileId);
+      testOperations(ownerUser2Api, RoleState.NONE, profileId);
```

Comment:
Just curious - why the switch to owner2?

Reply:
User1 is still in the admins group (dumbledore) so I wanted to use a user that explicitly wasn't in the admins group